src/disco/store.js

```diff
@@ -0,0 +1,11 @@
+import { applyMiddleware, createStore as _createStore, combineReducers } from 'redux';
+import { reducer as reduxAsyncConnect } from 'redux-async-connect';
+import createLogger from 'redux-logger';
+
+export default function createStore(initialState = {}) {
```

Comment:
I think at some point we'll only want to include `createLogger()` when in development and maybe even only on the client (I find it really noisy but sometimes useful in the console). That might be a good time to figure out a good way of doing so.